Gemfile

```diff
@@ -36,6 +36,7 @@ group :test do
   gem "minitest-reporters"
   gem "mocha", require: false
   gem "rails-controller-testing"
+  gem "rspec-rails"
```

Comment:
Can you explain in the commit message why we're moving to using RSpec matchers?
I vaguely remember it was needed, but can't remember why.

Reply:
Removed 👍